docker/daemon_none.go

```diff
@@ -0,0 +1,19 @@
+// +build !daemon
```

Comment:
basically this file's older version was named `docker/client.go`: https://github.com/docker/docker/pull/13771/files#diff-210c31cd49f34ac9a1fcc99dd77d9030L1